app/components/object-select/component.js

```diff
@@ -1,12 +1,18 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  actions: {
+ actions: {
```

Comment:
I believe this white space was correct (2 space indent) before, can you double check and update?